resources/views/auth/login.blade.php

```diff
@@ -15,7 +15,7 @@
                             <label for="email" class="col-md-4 col-form-label text-md-right">{{ __('E-Mail Address') }}</label>
 
                             <div class="col-md-6">
-                                <input id="email" type="email" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ old('email') }}" required autocomplete="email" autofocus>
+                                <input id="email" type="text" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ old('email') }}" required autocomplete="email" autofocus>
```

Comment:
```suggestion
                                <input id="email" type="email" class="form-control @error('email') is-invalid @enderror" name="email" value="{{ old('email') }}" required autocomplete="email" autofocus>
```
Not sure why it got changed to text?

Reply:
Was changed to text to allow for username login as well as email. Sure I saw that the backend was in place for that somewhere but with a `type="email"` the input won't accept a username.

Reply:
Sounds good